src/FSLibrary/BenchmarkDaemonSet.fs

```diff
@@ -0,0 +1,280 @@
+// Copyright 2025 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module BenchmarkDaemonSet
+
+open k8s
+open k8s.Models
+open System
+open System.Collections.Generic
+open Logging
+open StellarKubeSpecs
+open StellarNetworkCfg
+open StellarNetworkDelays
+open StellarCoreCfg
```

Comment:
It looks like `k8s`, `System.Collections.Generic`, `Logging` and `StellarCoreCfg` are unused and can be removed as imports.